ibm/service/vpc/data_source_ibm_is_backup_policy_job.go

```diff
@@ -148,6 +149,78 @@ func DataSourceIBMIsBackupPolicyJob() *schema.Resource {
 							Computed:    true,
 							Description: "The unique user-defined name for this volume.",
 						},
+						"remote": &schema.Schema{
```

Comment:
I think also `backup_policy_plan` has `remote` information as per API

Reply:
Not part of this SRB, still added to PR as it is part of GA feature